CASSANDRA.md

```diff
@@ -307,9 +300,34 @@ object Phantom extends App {
 }
 ```
 
-Phantom allows the user certain level of composability, but it gets a bit verbose due to the nature of the DSL. 
+Phantom allows the user certain level of composability, but it gets a bit verbose due to the nature of the DSL.
+However, phantom does not encourage this inherent duplication of concern because it doesn't add any value
+to an application, and instead generally prefers to duplicate from the start.
```

Comment:
I'm not sure I follow this change. What do you mean by "this inherent duplication of concern"?